it/controllers/benefits/reimbursed/VouchersOrCreditCardsBenefitsControllerISpec.scala

```diff
@@ -0,0 +1,534 @@
+    val expectedHeading = "Did your client get vouchers or credit cards?"
+    val expectedErrorTitle = s"Error: $expectedTitle"
+    val expectedErrorText = "Select yes if your client got vouchers or credit cards"
+    val paragraph = "We only need to know about vouchers your client can exchange for:"
+  }
+
+  object CommonExpectedEN extends CommonExpectedResults {
+    val expectedCaption = s"Employment for 6 April ${taxYearEOY - 1} to 5 April $taxYearEOY"
+    val expectedButtonText = "Continue"
+    val yesText = "Yes"
+    val noText = "No"
+    val bullet1 = "goods and services"
+    val bullet2 = "a sum of money substantially less than the cost to provide them"
+  }
+
+  object CommonExpectedCY extends CommonExpectedResults {
+    val expectedCaption = s"Employment for 6 April ${taxYearEOY - 1} to 5 April $taxYearEOY"
+    val expectedButtonText = "Continue"
+    val yesText = "Yes"
+    val noText = "No"
+    val bullet1 = "goods and services"
+    val bullet2 = "a sum of money substantially less than the cost to provide them"
+  }
+
+  val userScenarios: Seq[UserScenario[CommonExpectedResults, SpecificExpectedResults]] = {
+    Seq(UserScenario(isWelsh = false, isAgent = false, CommonExpectedEN, Some(ExpectedIndividualEN)),
+      UserScenario(isWelsh = false, isAgent = true, CommonExpectedEN, Some(ExpectedAgentEN)),
+      UserScenario(isWelsh = true, isAgent = false, CommonExpectedCY, Some(ExpectedIndividualCY)),
+      UserScenario(isWelsh = true, isAgent = true, CommonExpectedCY, Some(ExpectedAgentCY)))
+  }
+
+  ".show" should {
+
+    import Selectors._
+
+    userScenarios.foreach { user =>
+
+      import user.commonExpectedResults._
+
+      s"language is ${welshTest(user.isWelsh)} and request is from an ${agentTest(user.isAgent)}" should {
+
+        "render 'vouchers or credit cards' page with the correct content with no pre-filling" which {
+          lazy val result: WSResponse = {
+            dropEmploymentDB()
+            userDataStub(userData(fullEmploymentsModel()), nino, taxYearEOY)
+            insertCyaData(employmentUserData(isPrior = true, cyaModel("employerName", hmrc = true,
+              benefits = Some(benefits(fullReimbursedCostsVouchersAndNonCashModel.copy(vouchersAndCreditCardsQuestion = None))))), userRequest)
+            authoriseAgentOrIndividual(user.isAgent)
+            urlGet(pageUrl(taxYearEOY), welsh = user.isWelsh,
+              headers = Seq(HeaderNames.COOKIE -> playSessionCookies(taxYearEOY)))
+          }
+
+          implicit def document: () => Document = () => Jsoup.parse(result.body)
+
+
+          "has an OK status" in {
+            result.status shouldBe OK
+          }
+
+          titleCheck(user.specificExpectedResults.get.expectedTitle)
+          h1Check(user.specificExpectedResults.get.expectedHeading)
+          textOnPageCheck(expectedCaption, captionSelector)
+          textOnPageCheck(user.specificExpectedResults.get.paragraph, paragraphSelector)
```

Comment:
need to add textOnPage checks for the bullet pointed text

Reply:
Updated as requested.